setup.py

```diff
@@ -1,3 +1,3 @@
-from setuptools import setup
+from setuptools import find_packages, setup
 
-setup()
+setup(include_package_data=True, packages=find_packages())
```

Comment:
I think this can also go in pyproject.toml, (or it become useless), no ?

Reply:
yes, however, I saw this listed as still "beta" in the documentation as of today, which is why I moved it back into setup.py. I did the same for my own projects 1-2 months ago. It's possible that it will just work and that we can ignore the beta note in the documentation. https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

Reply:
Fair enough, thank you for the explanation :)